coderd/insights_internal_test.go

```diff
@@ -148,3 +150,110 @@ func Test_parseInsightsStartAndEndTime(t *testing.T) {
 		})
 	}
 }
+
+func Test_parseInsightsInterval_week(t *testing.T) {
+	t.Parallel()
+
+	layout := insightsTimeLayout
+	now := time.Now().UTC()
```

Comment:
We should either validate that startTime and endTime are UTC in `parseInsightsInterval` or explicitly test with random timezones here.